src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowMarkdownCommand.cs

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Collections;
+using System.Collections.ObjectModel;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Management.Automation;
+using Microsoft.PowerShell.MarkdownRender;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// Show the VT100EncodedString or Html property of on console or show.
+    /// VT100EncodedString will be displayed on console.
+    /// Html will be displayed in default browser.
+    /// </summary>
+    [Cmdlet(
+        VerbsCommon.Show, "Markdown",
+        HelpUri = "TBD"
+    )]
+    [OutputType(typeof(string))]
+    public class ShowMarkdownCommand : PSCmdlet
+    {
+        /// <summary>
+        /// InputObject of type Microsoft.PowerShell.MarkdownRender.MarkdownInfo to display
+        /// </summary>
+        [ValidateNotNullOrEmpty]
+        [Parameter(Mandatory = true, ValueFromPipeline = true)]
+        public PSObject InputObject { get; set; }
+
+        /// <summary>
+        /// Switch to view Html in default browser.
+        /// </summary>
+        [Parameter()]
+        public SwitchParameter UseBrowser { get; set; }
+
+        private SteppablePipeline stepPipe;
+
+        /// <summary>
+        /// </summary>
+        protected override void BeginProcessing()
+        {
+            if(! this.MyInvocation.BoundParameters.ContainsKey("UseBrowser"))
+            {
+                // Since UseBrowser is not bound, we use proxy to Out-Default
+                stepPipe = ScriptBlock.Create(@"Microsoft.PowerShell.Core\Out-Default @PSBoundParameters").GetSteppablePipeline(this.MyInvocation.CommandOrigin);
+                stepPipe.Begin(this);
+            }
+        }
+
+        /// <summary>
+        /// Override ProcessRecord
+        /// </summary>
+        protected override void ProcessRecord()
+        {
+            Object inpObj = InputObject.BaseObject;
```

Comment:
The same - `PSObject.Base()`?

Reply:
Followd the same pattern as https://github.com/PowerShell/PowerShell/blob/master/src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs#L239